### DISCUSSION TITLE
Serial-numbered version for App Store version?

### DISCUSSION THREAD

**Discussion by hdfonts** [rsayfutdinov/Pikka#37] [Q&A] [answered]
I have the serial-numbered version of Pikka 2 and would like to upgrade it to the App Store version. I can't find an email address for the developer and their Twitter account doesn't accept messages. I saw that there was a new version today, but my version doesn't include any way to update it in the app, and the only version I can find to download takes me to the App Store version. Is there anyway to "trade" my version for the App Store version without having to pay for it again?

**Comment by rsayfutdinov** [comment, marked as answer]
Hi @hdfonts ,

Sorry for the late reply.
Here is the promo code for you: EJPP9ENNK9NL

**Comment by hdfonts** [comment]
Thank you!

James Harris

**Comment by OllieMengedoht** [comment]
Hi folks,
(probably after changing to a new MacBook Pro after eight years) I'm having the same problem. Have a serial but where to type it in? „restore purchase“ changes nothing.

In app Store the only possibility is „Open“, not Buy or anything.

Best wishes, Ollie

**Reply by OllieMengedoht** [reply]
Still nothing here. I'm waiting… There is no way I can insert my license code in the app and so my Pikka Pro is a non pro! :-(

**Comment by RBHannam** [comment]
Hi,I'm having the same problem. I have a licence number, but it doesn't work now that I have moved from a Mac Book to a Mac Mini.
Grateful for your advice.
Thanks
